CONTRIBUTING.md

```diff
@@ -245,6 +245,12 @@ docker run jinaai/jina:my-local-version check
 
 It prints a list of components the current version of Jina supports, and then exits. Make sure yours are not in red.
 
+You need to build a local docker image tagged jinaai/jina:test-pip for all the tests to run as in the CI, via: 
```

Comment:
I would move this before the instructions to run `pytests`

Reply:
Sure